chromeapp/test/scripts/dnssd/dns-packet-test.js

```diff
@@ -0,0 +1,201 @@
+'use strict';
```

Comment:
You need unit tests for parseResourceRecordsFromReader, covering the error cases too.